boilerplate/app/navigators/DemoNavigator.tsx

```diff
@@ -54,6 +56,17 @@ export function DemoNavigator() {
         }}
       />
 
+      <Tab.Screen
+        name="DemoList"
+        component={DemoListScreen}
+        options={{
+          tabBarLabel: translate("demoNavigator.listTab"),
+          tabBarIcon: ({ focused }) => (
+            <Icon icon="menu" style={{ tintColor: focused && colors.tint }} />
```

Comment:
we probably want a podcast icon from @JenFucci

Reply:
Good call.

Reply:
@jamonholmgren btw, Icon has convenience prop called `color` so you don't have to specify a style.

Reply:
Oops, I missed this @yulolimum , maybe you can fix in your PR